script/setup/critools-version

```diff
@@ -1 +1 @@
-v1.27.0
+master
```

Comment:
The contents of `critools-version` are passed to `git checkout` so a commit (e.g. `7c8d9e273f57cc929c1244b0b8ea19d1eaa3a861`) should work and be safer than the cutting edge `master` branch.